fluid_onnx/ops.py

```diff
@@ -387,7 +427,7 @@ def elementwise_ops(op_type, operator, block):
         pre_shape.extend(shape)
         pre_shape.extend(post_shape)
         final_shape = [i if i > 0 else 1 for i in pre_shape]
-        shape_name = outputs['Out'][0] + "@shape_var"
+        shape_name = _tmp_name(outputs['Out'][0], "@shape_var")
```

Comment:
Note: in many cases, a model (e.g. ERNIE) may apply the same operator to a variable multiple times. The temporary variable name generated by the operator should be unique, else the conversion will fail with the error shown below. I only changed a few cases I need, but supposedly this should be done widely for any operator creating tmp node names.

```
The operator sets to run test case.
{'scale', 'reshape2', 'dropout', 'gelu', 'tanh', 'softmax', 'layer_norm', 'lookup_table', 'stack', 'mul', 'transpose2', 'slice', 'elementwise_add', 'matmul'}
Warning: Scale was a removed experimental ops. In the future, we may directly reject this operator. Please update your model as soon as possible.
Traceback (most recent call last):
  File "fluid_onnx/fluid_to_onnx.py", line 234, in <module>
    main()
  File "fluid_onnx/fluid_to_onnx.py", line 230, in main
    convert(args)
  File "fluid_onnx/fluid_to_onnx.py", line 194, in convert
    checker.check_model(onnx_model)
  File "/Users/winstonq/.tox/paddle/lib/python3.7/site-packages/onnx/checker.py", line 91, in check_model
    C.check_model(model.SerializeToString())
onnx.onnx_cpp2py_export.checker.ValidationError: Graph must be in single static assignment (SSA) form, however 'dropout_0.tmp_0@flatten_0' has been used as output names multip
```